### ISSUE TITLE
[promtail] Promtail Helm chart 6.14.2 missing from index

### ISSUE THREAD

**Issue by mprimeaux** [grafana/helm-charts#2603]
Promtail 6.14.2 is [the latest version](https://github.com/grafana/helm-charts/blob/c539d6a92953ee751f6ffe63640fd738c371074e/charts/promtail/Chart.yaml#L6), however it is missing from the index.

```shell
❯ helm repo add grafana https://grafana.github.io/helm-charts
"grafana" has been added to your repositories

❯ helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "grafana" chart repository

❯ helm search repo promtail
NAME            	CHART VERSION	APP VERSION	DESCRIPTION
grafana/promtail	6.14.1       	2.8.3      	Promtail is an agent which ships the contents o...
```

Expected behavior is to see promtail 6.14.1.

**Comment by tback** [comment]
duplicate of #2591, hopefully fixed in #2592

**Comment by mprimeaux** [comment]
Thanks. I’ll close this issue and monitor the other one:
